src/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const info = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <ul>
+        {info.map((element, index) => (
+          <li key={index}>{element}</li>
+        ))}
+      </ul>
+    </footer>
```

Comment:
This is nice! What if we received the info via props? Wouldn't make this component more dynamic?